### PR TITLE
chore: add maskito and new textfield example

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

File: projects/demo/src/modules/components/textfield/examples/4/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<tui-textfield>
+    <strong>π</strong>
+    -value
+
+    <input
+        placeholder="3,14"
+        tuiTextfield
+        [maskito]="options"
+        [(ngModel)]="value"
+    />
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/components/textfield/examples/4/index.html:10]: example doesn't work properly
caret jumps
backspace doesn't work

https://github.com/user-attachments/assets/94e3dd1c-d49b-4f30-9f19-289fd26dfaa3

File: projects/demo/src/modules/components/textfield/examples/4/index.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    imports: [FormsModule, MaskitoDirective, TuiIcon, TuiTextfield, TuiTooltip],
+    templateUrl: './index.html',
+    styleUrls: ['./index.less'],
+    encapsulation,
+    changeDetection,
+})
+export default class Example {
+    protected value = Math.PI.toFixed(48);
+    protected readonly options = maskitoNumberOptionsGenerator({
+        decimalSeparator: ', ',
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/components/textfield/examples/4/index.ts:21]: decimal separator should not be more than 1 character, I think that's the problem with the example

File: projects/demo/src/modules/components/textfield/examples/4/index.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {MaskitoDirective} from '@maskito/angular';
+import {maskitoNumberOptionsGenerator} from '@maskito/kit';
+import {TuiIcon, TuiTextfield} from '@taiga-ui/core';
+import {TuiTooltip} from '@taiga-ui/kit';
+
+@Component({
+    standalone: true,
+    imports: [FormsModule, MaskitoDirective, TuiIcon, TuiTextfield, TuiTooltip],
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/components/textfield/examples/4/index.ts:12]: TuiTooltip is unused?

File: projects/demo/src/modules/components/textfield/examples/4/index.ts (hunk 2 above)
Comment (splincode) [reply projects/demo/src/modules/components/textfield/examples/4/index.ts]: fixed

File: projects/demo/src/modules/components/textfield/examples/4/index.ts (hunk 3 above)
Comment (splincode) [reply projects/demo/src/modules/components/textfield/examples/4/index.ts]: fixed

**Review by vladimirpotekhin — APPROVED** [review]

**Review by mdlufy — APPROVED** [review]

File: projects/demo/src/modules/components/textfield/index.html [hunk 4]
```diff
@@ -15,6 +15,7 @@
             *ngFor="let example of examples; let index = index"
             [component]="index + 1 | tuiComponent"
             [content]="index + 1 | tuiExample"
+            [fullsize]="index === 3"
```
Comment (waterplea) [inline projects/demo/src/modules/components/textfield/index.html:18]: Maybe make it short instead? Long inputs do not look that good, there's no need for such a big number

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/components/textfield/index.html (hunk 4 above)
Comment (splincode) [reply projects/demo/src/modules/components/textfield/index.html]: fixed

**Pull request merged**